scripts/build_docker_image.sh

```diff
@@ -27,10 +27,10 @@ unzip .layers/datadog_extension.zip
 
 # Build the image, tagged with the version
 echo "Building the Docker image"
```

Comment:
It looks like this script now builds *and* pushes and the Docker image(s), I would rename it to reflect that, and also maybe update this `echo` message to make that more clear